tests/test_setup_command.py

```diff
@@ -1,4 +1,27 @@
+import tomlkit
+
+
+def check_result(result, reference, prefix="running pyproject\n"):
+    """
+    Check the result succeeded, and matches the expected output.
+    """
+    assert result.returncode == 0
+    assert result.stdout.startswith(prefix)
+
+    # Parse the reference in case of formatting differences
+    reference_parsed = tomlkit.parse(reference)
```

Comment:
Oh by the way: tomlkit's documentation says that it's a style-preserving TOML library, meant to preserve indentation, comments, etc. That's part of why I chose to use it over `tomli`, so we can get output with predictable formatting. So I think we shouldn't have to worry that much about formatting differences - or in other words, if the formatting does come out different, it means that part of the system is actually not doing what it's supposed to do, it wouldn't just be a random fluke. But we can always worry about that later.